webapp-client/src/stylesheets/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Is Item.css an empty file?

Reply:
Yeah Item.css in this PR is, but won't be in next PRs